Dockerfile

```diff
@@ -16,37 +138,27 @@ RUN yum --assumeyes install \
     python-pip \
     python3-requests-kerberos \
     rsync \
-    sshuttle \
```

Comment:
@mbarnes is sshuttle used for backplane login?

Reply:
Yes sshuttle is used for backplane. (Sorry I know you didn't ask me)

Reply:
Ah, good catch.  I guess ideally that'd be installed with backplane, but I'm not sure why I removed it here.  I'll fix that.